tests/cases/fourslash/server/pasteEdits_knownSourceFile.ts

```diff
@@ -0,0 +1,47 @@
+/// <reference path="../fourslash.ts" />
+
+// @Filename: /target.ts
+//// export const tt = 2;
+//// [|function f();|]
+//// const p = 1;
+
+// @Filename: /file1.ts
+////export const b = 2;
+
+// @Filename: /file2.ts
+////import { b } from './file1';
+////const a = 1;
+////[|const c = a + b;
+////const t = 9;|]
+
+// @Filename: /tsconfig.json
+////{ "files": ["file1.ts", "file2.ts", "target.ts"] }
+
+const range = test.ranges();
+const t = range[0];
+format.setOption("insertSpaceAfterSemicolonInForStatements", true);

```

Comment:
What is this for?

Reply:
I get an extra symbol in the server tests if  I don't use this 
<img width="173" alt="image" src="https://github.com/microsoft/TypeScript/assets/108360753/8f69ccc2-bf3b-4cf5-8545-6bfa36d7ae6e">

Reply:
I see, I'll take a look at this.

Reply:
Here, `TestState.formatCodeSetting` had  `newLineCharacter` as `\n` but `ProjectService.hostConfiguration.formatCodeOption` would have `newLineCharacter` as `\r\n`. So adding `this.setFormattingOptions(formatOptions);` in client.ts made it `\n` again.